projects/eval-adxl355-pmdz/sdp-ck1z.ioc

```diff
@@ -0,0 +1,147 @@
+#MicroXplorer Configuration settings - do not modify
```

Comment:
do we need this file added?

Reply:
yes i think we need it for stm32 builds
if we had a repository with such files hosted somewhere, we wouldn't need it, but since we don't...